packages/core-components/src/types.ts

```diff
@@ -0,0 +1,16 @@
+export type ButtonTypes =
```

Comment:
Just an FYI too - since these are all source files we'll need to include a [legal header](https://github.com/okta/okta-auth-js/blob/master/lib/OktaAuthBase.ts#L1-L11).